corehq/apps/reports/standard/cases/utils.py

```diff
@@ -171,3 +177,12 @@ def _get_location_accessible_ids(request):
 def query_location_restricted_cases(query, request):
     accessible_ids = _get_location_accessible_ids(request)
     return query.OR(case_es.owner(accessible_ids))
+
+
+def get_most_recent_case_type(domain):
```

Comment:
How about `get_most_recently_modified_case_type`? What does it mean for a case type to be modified, anyway?

Reply:
this is the most recently submitted case type to use as the default for the report because when dev and I talked he decided it was best to force a user to pick a specific case type, and this was my thought of the most intelligent way to default to a case type before the user goes ahead and modifies the filter and then that value is cached

Reply:
Interesting, that does sound pretty reasonable, it'll at least point the user towards data that's changing. It might be weird for domains with a bunch of case types, some more important than others, but if I remember right most SaaS domains only have one or two case types.

I do think it'll be difficult for us to get this right for every use case, especially for complicated domains, so this seems like a good place to make the user's most recent selection sticky on the client side. (Future suggestion, not for this PR.)

On the naming front, how about `get_type_of_most_recent_case`?